.github/workflows/Build.yaml

```diff
@@ -193,9 +192,41 @@ jobs:
           heap-size: 600M
           script: ./gradlew connectedDemoDebugAndroidTest --daemon
 
+      - name: Run local tests (including Roborazzi) for the combined coverage report (only API 30)
+        if: matrix.api-level == 30
+        # There is no need to verify Roborazzi tests to generate coverage.
+        run: ./gradlew testDemoDebugUnitTest -Proborazzi.test.verify=false # Add Prod if we ever add JVM tests for prod
+
+      # Add `createProdDebugUnitTestCoverageReport` if we ever add JVM tests for prod
+      - name: Generate coverage reports for Debug variants (only API 30)
+        if: matrix.api-level == 30
+        run: ./gradlew createDemoDebugCombinedCoverageReport
+        
       - name: Upload test reports
         if: always()
         uses: actions/upload-artifact@v4
         with:
           name: test-reports-${{ matrix.api-level }}
           path: '**/build/reports/androidTests'
+
+      - name: Display local test coverage (only API 30)
+        if: matrix.api-level == 30
+        id: jacoco
+        uses: madrapps/jacoco-report@v1.6.1
+        with:
+          title: Combined test coverage report
+          min-coverage-overall: 40
+          min-coverage-changed-files: 60
+          paths: |
+            ${{ github.workspace }}/**/build/reports/jacoco/**/*Report.xml
```

Comment:
Interesting, so this does the combining at the GHA level, not at the Gradle level? In other words, there isn't a local way to generate a combined coverage report from all modules.

Reply:
There are plugins and snippets that get you [there](https://github.com/SurpSG/jacoco-gradle-plugin-merge-coverage/blob/master/build.gradle#L28C1-L41C2), but a combined (jvm+device), combined (across modules) report is out of the scope of this PR 😁 I'll file a FR.